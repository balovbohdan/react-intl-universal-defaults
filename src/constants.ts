export const ERRORS = {
  NOT_INITIALIZED: `
    Please, initialize react-intl-universal-defaults!
    Doc: https://github.com/balovbohdan/react-intl-universal-defaults
  `,
};
