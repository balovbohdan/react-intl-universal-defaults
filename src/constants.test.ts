import * as constants from './constants';

describe('constants', () => {
  test('ERRORS', () => {
    expect(constants.ERRORS).toMatchSnapshot();
  });
});
