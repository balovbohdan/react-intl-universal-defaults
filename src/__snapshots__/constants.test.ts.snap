// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants ERRORS 1`] = `
Object {
  "NOT_INITIALIZED": "
    Please, initialize react-intl-universal-defaults!
    Doc: https://github.com/balovbohdan/react-intl-universal-defaults
  ",
}
`;
